examples/platform/linux/CommissionerMain.cpp

```diff
@@ -150,6 +190,9 @@ CHIP_ERROR InitCommissioner(uint16_t commissionerPort, uint16_t udcListenPort)
     const Credentials::AttestationTrustStore * testingRootStore = Credentials::GetTestAttestationTrustStore();
     SetDeviceAttestationVerifier(GetDefaultDACVerifier(testingRootStore));
 
+    // Uncomment the following line to perform DAC verification during NOC chain generation
+    // SetDeviceAttestationVerifier(&gPartialDACVerifier);
```

Comment:
This requires forking common code. Could this not be enabled/disabled with a dynamic setter?

Reply:
I think a lot of the objects in CommissionerMain used for commissioning will need to be overridden by specific implementations. I suggest moving this to a separate "cleanup" issue.

Reply:
https://github.com/project-chip/connectedhomeip/issues/22459